tests/kas_with_instance.spec.ts

```diff
@@ -177,3 +193,43 @@ test('create and delete a Kafka Topic', async ({ page }) => {
   await createKafkaTopic(page, testTopicName);
   await deleteKafkaTopic(page, testTopicName);
 });
+
+// test_6acl.py test_kafka_create_consumer_group_and_check_dashboard
+test('create consumer group and check dashboard', async ({ page }) => {
```

Comment:
I wonder if this test shouldn't be part of `kas_messaging.spec.ts` as it contains basically the same steps so we will save some time...on the other hand, this is probably more accurate based on the purpose from IQE. WDYT?

Reply:
There is the messaging part of the test so yes, we could move to `kas_messaging.spec.ts`